pinot-integration-test-base/src/test/java/org/apache/pinot/integration/tests/ClusterTest.java

```diff
@@ -572,4 +573,10 @@ public Object[][] useV2QueryEngine() {
         {true}
     };
   }
+
+  protected void notSupportedInV2() {
```

Comment:
per discussion with @gortiz This was added to skip V2 when there's test failure. however some test cases actually sequentially ran though the queries. so if the earlier one fails. there's no way to capture the latter ones. 

we should switch to a provider pattern otherwise the sequential query run will not capture enough